source/content/symlinks-assumed-write-access.md

```diff
@@ -43,10 +43,10 @@ The following is for Mac and Linux only. Windows users may refer to Microsoft do
 1. Move the directory you want to replace with a symlink. This serves to both back up any data that may otherwise be lost, and to prevent the symlink from being nested inside the existing directory:
 
     ```bash{promptUser: user}
-    mv ./wp-content/path/plugin-expects-write-to ~/backups/
+    mv ./wp-content/path/plugin-expects-write-to $_ENV['HOME']/tmp/backups
     ```
 
-    The command above moves the directory to a folder named `backups` in your home directory, `~/`. Replace this with your preferred backup location.
+    The command above moves the directory to the environment's home directory `/tmp/backups`. Replace this with your preferred backup location. Keep in mind that due to Pantheon's [container infrastructure](/application-containers), bindings may change and the only persistent directories that can be writen to are `/files` and `/code`.
```

Comment:
Again, this is a local backup, not one on the platform.

Reply:
I'm reexamining what you're pointing out in the first comment, to find a way to rework that to reflect a method that works locally and will continue working once the code is pushed.
For this one though, I think the statement is true- that is what the command above is doing, and if their `backups` directory of choice is within on directory that doesn't persist, it's worth noting (and I think was the source of the issue)

Reply:
@EdwardAngert disagree. See [L35-l40](https://github.com/pantheon-systems/documentation/pull/5808/files#diff-f31d3c5f73c5c66b40d1eb11b3624597R35-R40), and note that this whole section is scoped to a local clone of the code base on the user's computer. So saying "the environment's home directory" is false. Saying `/tmp/backups` is wrong for all cases that aren't on the platform, unless the reader has a highly unusual desktop configuration where their home directory is set to `/` (and if that were the case, they'd have bigger problems). 

While the information on persistent directories in the platform is correct, it's irrelevant to the task at hand, which is making a _local_ backup of the content being replaced in code by a symlink.